spec/test_app/app/routing/bulletins_routing_spec.rb

```diff
@@ -36,5 +36,15 @@
                                         format: 'json')
     end
   end
+
+  describe 'GET /v1/bulletins/sign' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.